src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -190,16 +190,18 @@ def load_arguments(self, _):
         c.argument('vnet_subnet_id')
         c.argument('workspace_resource_id')
         c.argument('skip_subnet_role_assignment', action='store_true')
+        c.argument('api_server_authorized_ip_ranges', type=str, validator=validate_ip_ranges_on_create)
```

Comment:
Interesting, I was told we would always be defaulting to standard now. I will update the validator to only check when k8 version >= 1.13.0. 

I want to keep it in the validator though instead of the core logic - I think the separate will lead to a cleaner, more re-useable code base

Reply:
Actually I think this is still right. The validation only disallows basic LB with IPWL which is an illegal operation. If IPWL isn't provided then basic will be rightfully allowed

Reply:
Talked offline. I have changed the validation code to not kick out basic LB until after the LB has been set in custom.py. I have also moved the code to determine the LB type into a helper function for easy of reading / re-use